make_queue/static/make_queue/js/stream.js

```diff
@@ -30,7 +30,7 @@ $("html").keydown(function (event) {
     }
 });
 
-var closeFullscreen = function () {
+let closeFullscreen = function () {
```

Comment:
This will be redefined for each stream, so `let` will result in a Javascript error. See the `/reservation/` page as an example.